tests/test_data/rezeptwelt.testhtml

```diff
@@ -0,0 +1,2723 @@
+<!DOCTYPE html>
+<html xmlns:fb="http://www.facebook.com/2008/fbml" xmlns:og="http://opengraphprotocol.org/schema/">
+<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article# video: http://ogp.me/ns/video#">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta property="fb:app_id" content="159832351186853"/>
+        <title>Nudelsalat von Thermomix Rezeptentwicklung. Ein Thermomix ® Rezept aus der Kategorie sonstige Hauptgerichte auf www.rezeptwelt.de, der Thermomix ® Community.</title>
+<meta name="twitter:card" content="summary_large_image"/>
+<meta property="og:title" content="Nudelsalat"/>
```

Comment:
What do you think about retrieving a shorter-format recipe title from a field like this one?

(that could make the data extraction less fragile; for example if the site changes their SEO strategy)